bin/cyhy-import

```diff
@@ -146,6 +163,9 @@ def main():
 
     if not success:
         sys.exit(-1)
+    else:
+        print("Success.")
```

Comment:
```suggestion

    print("Success.")
```

Reply:
Oops, this statement can actually be removed. I only added it in when I commented out the actual importing and adding of IPs when I was doing testing so I can know whether the import would have been successful or not. If you all would like to keep the success message then that's cool, but it wasn't there before and isn't needed for this change

Reply:
I'm fine with removing if it's not something your team needs.